test/integration/utils/test_xml_validation.py

```diff
@@ -68,5 +68,16 @@ def test_validate_xml_empty_label() -> None:
         validate_xml_file(input_file="testdata/invalid-testdata/xml-data/empty-label.xml")
 
 
+def test_validate_xml_duplicate_license_copyright_id() -> None:
+    expected_msg = regex.escape(
+        "The XML file cannot be uploaded due to the following validation error(s):\n    "
+        "Line 11: Element 'copyright', attribute 'id': 'copyright_id' is not a valid value of the atomic type 'xs:ID'."
```

Comment:
Make it consistent with the line below

```suggestion
        "The XML file cannot be uploaded due to the following validation error(s):"
        "\n    Line 11: Element 'copyright', attribute 'id': 'copyright_id' is not a valid value of the atomic type 'xs:ID'."
```